SETUP.md

```diff
@@ -12,6 +12,7 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
 1. `rbenv rehash`
 1. `cd code-dot-org`
 1. `bundle install`
+1. `cp locals.yml.default locals.yml`
```

Comment:
perhaps `bundle install` should perform this step automatically if locals.yml is not present?